docs/source/usage/plugins.rst

```diff
@@ -61,6 +61,24 @@ Examples
 * ``42,30:50:10``: at steps 30 40 42 50 84 126 168 ...
 * ``5,10``: at steps 0 5 10 15 20 25 ... (only executed once per step in overlapping intervals)
 
+Python postprocessing
+=====================
+
+In order to further work with the data produced by a plugin during a simulation run, PIConGPU provides
+python tools that can be used for reading the data and for visualization
+(at the moment only for energy_histogram, PNG and phase_space).
+
+
+They can be found under ``lib/python/picongpu/``.
+It is our goal to provide three modules for each plugin to make postprocessing as convenient as possible:
```

Comment:
provide *at least*

the thing is: every plugin can have multiple data readers and multiple data visualization methods. For example, the energy histogram can analyze
- a histogram
- an integrated charge per opening angle in a detector for a range in the histogram
- cut-off energies
- ...